README.md

```diff
@@ -44,7 +44,7 @@
   - Never nest ternary statements
   - Wrap the condition in ()
   - Must be contained on either one or three lines
-  - Never use the shorthand version `?:`
+  - The shorthand `?:` may be used as a substitute for the expression `empty($a) ? $a : $b`
```

Comment:
`! empty()`

but that isn't completely true since `$a` has to be defined

Reply:
I didn't mean to imply that, but will add a note.